explanations/dispatchers.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    a = jnp.linspace(0, 1, 11)
</pre><p> </p><p>Would we see just the decimals then? I find the many digits a bit distracting...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/lcm/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>